asterisk/rootfs/etc/services.d/discovery/run

```diff
@@ -3,22 +3,27 @@
 # Home Assistant Community Add-on: Asterisk
 # Send Discovery information to Home Assistant
 # ==============================================================================
-declare config
 
-# Wait for motionEye to start before continuing
-bashio::net.wait_for 8089
+# shellcheck shell=bash
+
+# Wait for Asterisk to become available before continuing
+bashio::net.wait_for 8088
 
 # Create discovery config payload for Home Assistant
 config=$(bashio::var.json \
     host  "127.0.0.1" \
-    port "8089" \
+    port "5038" \
```

Comment:
@TECH7Fox isn't this the correct port you meant to send?

Reply:
It is, thanks.